2020-12-21 17:49:16,653 - D:/SOFTWARE_DATA/PycharmProject/PycharmProjects/untitled/autotesttv/report/test1.py[line:38] - ERROR: error
2020-12-21 17:50:17,102 - D:/SOFTWARE_DATA/PycharmProject/PycharmProjects/untitled/autotesttv/report/test1.py[line:38] - ERROR: error
